### PR TITLE
Fixed postprocessor for top K

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Fixed postprocessing for operations with several outputs

### Tickets:
 - 71691

### CODE REVIEW

File: ngraph/core/src/preprocess/preprocess_steps_impl.cpp [hunk 1]
```diff
@@ -385,7 +385,7 @@ void PostStepsList::add_convert_impl(const element::Type& type) {
         if (t == element::Type{}) {
             t = ctxt.target_element_type();
         }
-        if (t == node.get_node()->get_element_type()) {
+        if (t == node.get_node()->get_output_element_type(node.get_index())) {
```
Comment (ilya-lavrenov) [inline ngraph/core/src/preprocess/preprocess_steps_impl.cpp:388]: do we have more such places in prepro?

Comment (ilyachur) [reply ngraph/core/src/preprocess/preprocess_steps_impl.cpp]: I checked and didn't find any other names.
@nosovmik Please correct me if I am wrong

File: ngraph/core/src/preprocess/preprocess_steps_impl.cpp [hunk 2]
```diff
@@ -385,7 +385,7 @@ void PostStepsList::add_convert_impl(const element::Type& type) {
         if (t == element::Type{}) {
             t = ctxt.target_element_type();
         }
-        if (t == node.get_node()->get_element_type()) {
+        if (t == node.get_element_type()) {
```
Comment (nosovmik) [inline ngraph/core/src/preprocess/preprocess_steps_impl.cpp:388]: Yes, this way is better, thanks a lot for the findings

**Review by nosovmik — APPROVED** [review]

File: ngraph/core/src/preprocess/preprocess_steps_impl.cpp (hunk 1 above)
Comment (nosovmik) [reply ngraph/core/src/preprocess/preprocess_steps_impl.cpp]: yes, I used `node.get_element_type()` everywhere except this place. Will also implement unit test for this

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**